src/utils/__tests__/PaymentConverter.tests.ts

```diff
@@ -0,0 +1,393 @@
+          codiceIdentificativoUnivoco: "001"
+        },
+        denominazioneBeneficiario: "BANCA01",
+        codiceUnitOperBeneficiario: "01",
+        denomUnitOperBeneficiario: "DENOM01",
+        indirizzoBeneficiario: "VIAROMA",
+        civicoBeneficiario: "01",
+        capBeneficiario: "00000",
+        localitaBeneficiario: "ROMA",
+        provinciaBeneficiario: "ROMA",
+        nazioneBeneficiario: "IT"
+      },
+      credenzialiPagatore: "NOMECOGNOME",
+      causaleVersamento: "CAUSALE01",
+      spezzoniCausaleVersamento: [
+        {
+          spezzoneCausaleVersamento: "SPEZZONE1",
+          spezzoneStrutturatoCausaleVersamento: {
+            causaleSpezzone: "CAUSALE01",
+            importoSpezzone: 0
+          }
+        }
+      ]
+    }
+  }
+};
+
+const anAttivaRPTOutputKoIban: InodoAttivaRPTOutput = {
+  nodoAttivaRPTRisposta: {
+    fault: {
+      faultCode: "01",
+      faultString: "FAULTSTRING",
+      id: "ID01",
+      description: "FAULTDESCRIPTION",
+      serial: 1
+    },
+    esito: PPTPortTypes.Esito.KO,
+    datiPagamentoPA: {
+      importoSingoloVersamento: 99.05,
+      ibanAccredito: "XXX",
+      bicAccredito: "BPPIITRR",
+      enteBeneficiario: {
+        identificativoUnivocoBeneficiario: {
+          tipoIdentificativoUnivoco: "G",
+          codiceIdentificativoUnivoco: "001"
+        },
+        denominazioneBeneficiario: "BANCA01",
+        codiceUnitOperBeneficiario: "01",
+        denomUnitOperBeneficiario: "DENOM01",
+        indirizzoBeneficiario: "VIAROMA",
+        civicoBeneficiario: "01",
+        capBeneficiario: "00000",
+        localitaBeneficiario: "ROMA",
+        provinciaBeneficiario: "ROMA",
+        nazioneBeneficiario: "IT"
+      },
+      credenzialiPagatore: "NOMECOGNOME",
+      causaleVersamento: "CAUSALE01",
+      spezzoniCausaleVersamento: [
+        {
+          spezzoneCausaleVersamento: "SPEZZONE1",
+          spezzoneStrutturatoCausaleVersamento: {
+            causaleSpezzone: "CAUSALE01",
+            importoSpezzone: 99.05
+          }
+        }
+      ]
+    }
+  }
+};
+
+const aPaymentCheckRequest: PaymentsCheckRequest = {
+  codiceIdRPT: {
+    CF: "DVCMCD99D30E611V" as FiscalCode,
+    AuxDigit: "0",
+    CodIUV: "010101010101010" as IUV,
+    CodStazPA: "22"
+  }
+};
+
+const aPaymentActivationRequest: PaymentsActivationRequest = {
+  codiceIdRPT: {
+    CF: "DVCMCD99D30E611V" as FiscalCode,
+    AuxDigit: "0",
+    CodIUV: "010101010101010" as IUV,
+    CodStazPA: "22"
+  },
+  importoSingoloVersamento: 99.05 as Importo,
+  codiceContestoPagamento: "12345"
+};
+
+const aCodiceContestoPagamento: CodiceContestoPagamento = "12345" as WithinRangeString<
+  1,
+  35
+>;
+
+const aConfig = Configuration.decode(CONFIG).getOrElseL(errors => {
+  throw Error(`Invalid configuration: ${reporters.readableReport(errors)}`);
+});
+
+describe(" Test getPaymentsCheckRequestPagoPa to convert PaymentCheckRequest to InodoVerificaRPTInput", () => {
```

Comment:
you want also to check that the functions below don't just return a `Right` but also return the expected value

Reply:
also, `describe` should be passed the "thing" you're testing (in this case you're testing `getPaymentsCheckRequestPagoPa`) and the `it`s should describe the expected behavior - the format/naming of the tests should be:

```ts
describe("getPaymentsCheckRequestPagoPa", () => {
  it("should convert PaymentCheckRequest to InodoVerificaRPTInput", () => { ... }).
}
```